app/jobs/refresh_templates_rendering_statuses_job.rb

```diff
@@ -0,0 +1,36 @@
+class RefreshTemplatesRenderingStatusesJob < ApplicationJob
+  around_perform do |job, block|
+    block.call
+
+    self.class.queue
+  rescue StandardError => e
+    Foreman::Logging.logger('background').error("#{self.class} error: #{e} \n#{e.backtrace.join("\n")}")
```

Comment:
This logger needs to be defined in `logging.rb` and added to the YAML example setting and installer.